src/comgr.cpp

```diff
@@ -61,27 +61,27 @@
 /// More info at https://github.com/ROCmSoftwarePlatform/MIOpen/issues/1257.
 #define WORKAROUND_ISSUE_1257 (HIP_PACKAGE_VERSION_FLAT >= 4003021331ULL)
 
-MIOPEN_DECLARE_ENV_VAR(MIOPEN_DEBUG_COMGR_LOG_CALLS)
-MIOPEN_DECLARE_ENV_VAR(MIOPEN_DEBUG_COMGR_LOG_SOURCE_NAMES)
+MIOPEN_DECLARE_ENV_VAR(MIOPEN_DEBUG_COMGR_LOG_CALLS, bool, false)
+MIOPEN_DECLARE_ENV_VAR(MIOPEN_DEBUG_COMGR_LOG_SOURCE_NAMES, bool, false)
 
 /// 0: Off.
 /// 1: Logs each option on a separate line.
 /// 2: Logs all options altogether, on single line.
-MIOPEN_DECLARE_ENV_VAR(MIOPEN_DEBUG_COMGR_LOG_OPTIONS)
+MIOPEN_DECLARE_ENV_VAR(MIOPEN_DEBUG_COMGR_LOG_OPTIONS, uint64_t, 0)
```

Comment:
And I'm curios: why not define an enum or uint8_t type for variables like this?

Reply:
Creating multiple types for enums seems inadvisable.
It's possible to add more integer types, but this would increase code complexity (more template definitions) and only provide minor runtime memory reduction.